dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/api/Connection.java

```diff
@@ -103,14 +106,19 @@ private Bootstrap create() {
             @Override
             protected void initChannel(SocketChannel ch) {
                 ch.attr(CONNECTION).set(Connection.this);
-                // TODO support SSL
+
+                SslContext sslContext = SslContexts.buildClientSslContext(url);
```

Comment:
SSL context shoule only be created when URL enables it

Reply:
done